wire-library/wire-compiler/src/main/java/com/squareup/wire/schema/fileSystems.kt

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
Just confirming you want the file name to start with a lower case

Reply:
Yeah, I think the scheme I observed elsewhere is lowerCamel – no primary top-level class, UpperCamel – primary top-level class